metadata-protobuf/proto/WorkingGroups.proto

```diff
@@ -57,3 +57,14 @@ message WorkingGroupMetadataAction {
     RemoveUpcomingOpening remove_upcoming_opening = 3;
   }
 }
+
+message ModeratePost {
+  optional uint64 post_id = 1;
+  optional string rationale = 2;
```

Comment:
Any particular reason why these fields are marked `optional` and not `required`. If these fields were mandatory then you would not have to check the possibility of undefined in the [mappings](https://github.com/thesan/joystream/blob/cf06f473330110c157e5c18bc2886f3a00428b58/query-node/mappings/src/workingGroups.ts#L664-L665)

Reply:
Right, at least making `post_id` required makes more sense. But since AFAIK the runtime doesn't check `metadata`, the QN should still check whether they are defined